providers/github_test.go

```diff
@@ -104,8 +105,9 @@ func TestGitHubProviderGetEmailAddress(t *testing.T) {
 	bURL, _ := url.Parse(b.URL)
 	p := testGitHubProvider(bURL.Host)
 
+	ctx := context.TODO()
```

Comment:
Please do the others in this file too
```suggestion
	ctx := context.Background()
```